mapper/tedge_mapper/src/time_provider.rs

```diff
@@ -0,0 +1,25 @@
+use chrono::{DateTime, FixedOffset};
+
+pub type Timestamp = DateTime<FixedOffset>;
+
+pub trait TimeProvider {
```

Comment:
Just a reminder to use Albin's `Clock` trait. You can also add a `TestClock` to his `clock` or `time` crate, once merged.